tools/linter_lib/custom_check.py

```diff
@@ -769,7 +769,7 @@ def check_custom_checks_py():
         color = next(colors)
 
         for fn in by_lang['py']:
-            if 'custom_check.py' in fn:
+            if 'custom_check.py' or 'custom_rules.py' in fn:
```

Comment:
I think probably the right fix for these is to tweak the specific rules that fail to have an exclude and/or fix these lines; `custom_check.py` needs to be excluded unconditionally because by construction, it has lots of examples of bad code in it, but `custom_rules.py` shouldn't need to.  

```
(zulip-py3-venv) tabbott@coset:~/zulip$ ./tools/lint
py        | For scripts run as part of installer, cannot rely on typing existing; use `if False` workaround. at tools/zulint/custom_rules.py line 11:
py        | from typing import Any, Dict, List, Optional, Tuple, Iterable
Line too long (114) at tools/zulint/custom_rules.py line 174:              'description': "Use `#!/usr/bin/env foo` instead of `#!/path/foo` for interpreters other than sh."},
```

Reply:
Taken care of.